Spigot-Server-Patches/0267-More-Anvil-Events.patch

```diff
@@ -0,0 +1,42 @@
+From c9f275645f3615195fb55d228535e48521f61ee5 Mon Sep 17 00:00:00 2001
+From: BillyGalbreath <Blake.Galbreath@GMail.com>
+Date: Sun, 21 Jan 2018 05:09:49 -0600
+Subject: [PATCH] More Anvil Events
+
+
+diff --git a/src/main/java/net/minecraft/server/ContainerAnvil.java b/src/main/java/net/minecraft/server/ContainerAnvil.java
+index 16ec67569..7bcfd64e2 100644
+--- a/src/main/java/net/minecraft/server/ContainerAnvil.java
++++ b/src/main/java/net/minecraft/server/ContainerAnvil.java
+@@ -74,13 +74,22 @@ public class ContainerAnvil extends Container {
+                     int i = ((Integer) iblockdata.get(BlockAnvil.DAMAGE)).intValue();
+ 
+                     ++i;
+-                    if (i > 2) {
```

Comment:
Can we avoid the indentation change? Cant see context below, but could we return early if the damage event is cancelled?